### PR TITLE
Solution to Clone Problem

### PR DESCRIPTION
**Pull request by whilestevego**
😎 
image

### CODE REVIEW

**Comment by whilestevego** [comment]
Hello! Stuff!

**Comment by nm0s** [comment]
👍

File: clone/solution.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function clone (obj) {
```
Comment (kayshageorge) [inline clone/solution.js]: 💯

File: clone/solution.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  let newObj = {};
```
Comment (whilestevego) [inline clone/solution.js]: Please go with `newObject` instead of `new Obj`

**Comment by tomm-ylee** [comment]
Test

File: clone/solution.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+  for (let property in obj) {
+    newObj[property] = obj[property];
+  }
+  return newObj;
```
Comment (gustavoparolin) [inline clone/solution.js]: I'm missing comments

File: clone/solution.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+function clone (obj) {
+  let newObj = {};
+
+  for (let property in obj) {
+    newObj[property] = obj[property];
```
Comment (whilestevego) [inline clone/solution.js]: 👋

**Comment by hytonylee** [comment]
>

**Comment by phongpt1989** [comment]
asdfadf

**Comment by metika** [comment]
👍

**Pull request merged**